packages/builder/src/pages/builder/app/[application]/design/[screenId]/[componentId]/_components/Component/ComponentSettingsSection.svelte

```diff
@@ -19,6 +19,8 @@
   export let includeHidden = false
   export let tag
 
+  let sections = []
```

Comment:
Do we still need this change? I'm not sure what it does now that we've gone for this other approach to fix this issue.

Reply:
@aptkingston Whoops, nope, left over. Also removed some renaming changes that I'd made from the previous implementation that aren't strictly necessary now. 

Also just in case it comes up, I checked that no other consumers of this use the event based stuff before I removed it, also no other consumers used the "show" prop so they shouldn't be affected by the change from "show" to "initiallyShow"

If you could review this again when you get a chance please